### DISCUSSION TITLE
Can I use different units in LAMMPS like real instead of metal?

### DISCUSSION THREAD

**Discussion by saeedshahi** [deepmodeling/deepmd-kit#1262] [Q&A] [answered]
Hello all,

I have a couple of question

1. Can I do MD on NPT ensemble by using NN potential which is trained by an NVT ensemble dataset like you put on your website?
2. If I trained 64 water molecules by DeePMD and got a NN potential, can I use it for MD of 256 water molecules? I mean I don't need to do MD by the same number that I trained. right?
3. Can I use different units in LAMMPS like real instead of metal? what is the unit to be compatible with graph.pb?

Best,
Saeed

**Comment by njzjz** [comment, marked as answer]
1. Yes.
2. Yes.
3. Not supported yet.

**Comment by njzjz** [comment]
The support for `real` is added in #2775.
